Gordon360/Controllers/ProfilesController.cs

```diff
@@ -449,6 +450,37 @@ public async Task<ActionResult<StudentProfileViewModel>> UpdateMobilePhoneNumber
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update planned graduation year
+        /// </summary>
+        /// <param name="value">planned graduation year</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("planned_graduation_year")]
+        //
```

Comment:
Do you mean stateyourbusiness? I didn't add it here because I didn't see it on some other similar functions (e.g. update mobile phone number). Do you think we need it here?

Reply:
Correct, StateYourBusiness is the authorization handler.

You're correct, this actually doesn't need further authorization logic because it always operates on the currently authenticated user, just like the update mobile phone endpoint.